ush/parsing_ufs_configure.sh

```diff
@@ -77,6 +77,16 @@ if [[ "${cplwav}" = ".true." ]]; then
   local MULTIGRID="${waveMULTIGRID}"
   local WW3_user_sets_restname="false"
 
+  local WW3_user_histname="false"
+  local WW3_historync="false"
+  local WW3_restartnc="false" 
+  local WW3_restart_from_binary="true"
```

Comment:
The restart_from_binary option is only valid if you're also using restartnc. It allows you start from a binary but only write restarts in netcdf going forward.

If restartnc is false, then the option is not exercised.

Reply:
The PIO restart and binary restarts are not yet being used.  We are still using the inp file.  @sbanihash is working on moving to nml and then we can turn on this feature.  This was not done for this PR to expedite getting other fixes in.

Reply:
I understand that, but you're setting the restart_from_binary=true, which implies you're asking the code to do something. But this setting is not pertinent w/o using restartnc.

Reply:
changed